lib/modulesync/renderer.rb

```diff
@@ -10,8 +10,14 @@ def initialize(configs = {})
     end
 
     def self.build(from_erb_template)
```

Comment:
using this code and the tests packaged with this PR:
```ruby
    def self.build(template_file)
      erb_obj = ERB.new(File.read(template_file), nil, '-')
      erb_obj.filename = template_file.chomp('.erb')
      erb_obj.def_method(ForgeModuleFile, 'render()')
      erb_obj
    end
```

returns the following failures:
cucumber features/update.feature:628 # Scenario: Using .erb extension template but settings ommit .erb
cucumber features/update.feature:663 # Scenario: Using non-.erb extension template but settings use .erb

Please advise